app/.pylintrc

```diff
@@ -2,7 +2,8 @@
 disable=
     fixme,
     invalid-name,
-    missing-docstring
+    missing-docstring,
+    too-few-public-methods
```

Comment:
This should not be globally disabled. Having classes with only a few methods is a code smell (should be replaced with functions). In rare cases (e.g. models) it can be disabled manually.